### DISCUSSION TITLE
Getting 404 when trying to use streaming recognition mode

### DISCUSSION THREAD

**Discussion by 0x5F0887DF** [deepgram/community#209] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

I've used the test suite [client.py ](https://github.com/deepgram/streaming-test-suite/blob/main/client.py) to transcribe audio in streaming mode.

The problem is that DG API returns HTTP 404 error.
Could you please tell me what went wrong?

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

wss://api.deepgram.com/v1/listen?encoding=linear16&sample_rate=8000&channels=1

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

96e49290-d275-4281-9348-0f2fc2d3ccbd

**Comment by shirgoldbird** [comment]
Hi @0x5F0887DF! Could you verify the Python command you're using to run the test suite?

The [streaming test suite](https://github.com/deepgram/streaming-test-suite) contains two different client files—`client.py`, which is designed to connect to a local server for debugging purposes, and `test_suite.py`, which is designed to connect to Deepgram. 

While `client.py` can also connect to Deepgram if you modify the file (as described in [Step 3](https://github.com/deepgram/streaming-test-suite#3-streaming-other-audio) of the README), I'd suggest trying to run `test_suite.py` first to verify you can transcribe audio with Deepgram using that code.

**Reply by 0x5F0887DF** [reply]
Hi @shirgoldbird !
Thank you for your reply!
I've made modifications to the client.py to connect to the Deepgram API (not local server). I also imported logging to see websockets module messages. So it works as expected. I've provided url earlier.

I just can't understand why DG returns 404

**Reply by shirgoldbird** [reply]
@0x5F0887DF that's very odd! I just tried running `client.py` with line 20 uncommented and my API key added on line 28. Everything worked as expected and I received transcriptions from the Deepgram API. 

Could you post the diff of your local `client.py` vs what's on the main branch? Just wondering if any unintentional changes snuck in there.

**Reply by 0x5F0887DF** [reply, marked as answer]
I've found the trouble in my DNS settings.
Thank you for your help.
